docs/scripts/index.htm

```diff
@@ -45,11 +44,6 @@ <h2 id="toc">Table of Contents</h2>
   <li><a href="#Scripts_and_Functions_Forum">Archived Scripts and Functions Forum</a></li>
 </ul>
 
-<h2 id="NiftyWindows">NiftyWindows</h2>
-<p>Author: Enovatic-Solutions</p>
-<p>NiftyWindows gives you easy control of all basic window interactions such as dragging, resizing, maximizing, minimizing and closing. Its most powerful feature is activated by dragging with the right mouse button. Visualize each window divided into a virtual 9-cell grid with three columns and rows. The center cell is the largest one: you can grab and move a window around by clicking and holding it with the right mouse button. The other eight cells are used to resize a window in the same manner. NiftyWindows also offers snap-to-grid, "keep window aspect ratio", rolling up a window to its title bar, transparency control, and other useful shortcuts.</p>
-<p><a href="http://www.enovatic.org/products/niftywindows/introduction/">Download, introduction and more</a></p>
```

Comment:
👍 Looking forward to bring this back, once this is completed: https://github.com/ahkscript/awesome-AutoHotkey/issues/180#issuecomment-923114085

Reply:
I would not recommend investing time to bring the site back online because:

- you would have to ask the author/company for permission first
- the program/script itself doesn't work anymore either, as it tries to access the dubious site, which causes Windows security to terminate NiftyWindows

Reply:
Whoops... I only read this comment just now... https://github.com/ahkscript/NiftyWindows